packages/warp-cache/src/internal/cacheHttpClient.ts

```diff
@@ -511,6 +518,30 @@ export async function saveCache(
         ''
       break
     }
+
+    case 'azure_blob': {
+      if (!AzureSASURL) {
+        core.debug(`Azure SAS URL is not set. SAS URL: ${AzureSASURL}`)
+        throw new Error(
+          'Unable to upload cache to Azure Blob. SAS URL is not provided.'
+        )
+      }
+
+      core.debug('Uploading cache')
+      const blockBlobClient = new BlockBlobClient(AzureSASURL)
+      await blockBlobClient.uploadFile(archivePath, {
+        maxSingleShotSize: 10 * 1024 * 1024 // 10 MB
```

Comment:
Is 10 MB optimal?

Reply:
Should we increase or leave the default?

> If the size of the data is less than or equal to this value, it's uploaded in a single put rather than broken up into chunks. If the data is uploaded in a single shot, the block size is ignored. Default value is 256 MiB.
>
> _From https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blob-upload-javascript_

Reply:
Default value for parallel uploads is way higher than what we use across other providers. That's why it is overridden here.